src/components/dropdown.js

```diff
@@ -47,5 +52,9 @@ class FlavorForm extends React.Component {
     }
   }
 
+  const stateMappings = state => ({})
+  const actionMappings = {
+      dropDown
```

Comment:
Because this is a generic dropdown component designed for re-use, you probably want to avoid binding it to the Redux store. Higher level components (e.g., the order details page) can bind to the store state and actions, and lower level components like this can communicate with higher level components just through props.

This line is fine for this merge, but you'll definitely want to remove this redux mapping.